docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -70,17 +70,27 @@ export LDAP_PASSWORD="Your_LDAP_pw_here"
 
 === Configure input
 
+.Access to various APIs
 [source,console]
 ----
 export CLOUDSCALE_TOKEN=<cloudscale-api-token> # From https://control.cloudscale.ch/user/api-tokens
 export GITLAB_TOKEN=<gitlab-api-token> # From https://git.vshn.net/profile/personal_access_tokens
-export REGION=rma # rma or lpg (without the zone number)
+export COMMODORE_API_URL=<lieutenant-api-endpoint> # For example: https://api-int.syn.vshn.net
```

Comment:
We probably want to add a note here that the `COMMODORE_API_URL` *must* have a trailing slash for the subsequent `curl` commands to work.

Reply:
I'd rather add the slash to the commands...

Reply:
I upted for adding the `/`. That was anyway my initial intention but somehow got lost.

Reply:
Then we **must** ensure that there's no trailing slash here, otherwise Lieutenant API isn't happy either:

```shell
$ curl https://api.syn.vshn.net/healthz
ok
$ curl https://api.syn.vshn.net//healthz
{"reason":"Path was not found"}
```